app/assets/javascripts/comment.js

```diff
@@ -6,7 +6,6 @@
         $(this).find('#text-input').prop('disabled',false);
         $(this).find('#text-input').val('');
         $('#comments-container').append(xhr.responseText);
-        $('#comment-count')[0].innerHTML = parseInt($('#comment-count')[0].innerHTML, 10)+1;
```

Comment:
Removing this solves the issue but how the control is coming here as comment is not added?

Reply:
I can add a validation to verify when the comment isn't added and display an error, I'm going to add that :+1: 
Thank you @gauravano

Reply:
Should I add a new issue for this? @gauravano

Reply:
Actually, let's put a Javascript condition to disable the button if the comment body is empty. How does it sound?

Also, sorry for delay here.

Reply:
@ValeriaCobianISW regarding showing the error message in case of failure, we already have that but it needs some fix. Like, I tested commenting the empty body on production and count increased but when I went to dashboard this alert appeared 

![Screenshot from 2019-04-30 17-37-42](https://user-images.githubusercontent.com/20878070/56960581-f6a39100-6b6e-11e9-8c28-be5482983549.png)

may be we need to change it to flash.now or something. Would you like to fix this too?

Thanks!

Reply:
Sure, I would like to fix it, I'm going to open a new issue and work on it, thank you @gauravano :D